variables.tf

```diff
@@ -24,7 +24,7 @@ variable "github_repo" {
   type = object({
     repo         = string
     branches     = optional(list(string), [])
-    environments = optional(list(string), [])
+    environments = optional(list(string), ["*"])
```

Comment:
@niroz89 , made this default to `["*"]` because github action will get access denied when using environments